storage/src/commit_log/error.rs

```diff
@@ -0,0 +1,37 @@
+use failure::Fail;
+use std::io;
+
+#[derive(Debug, Fail)]
+pub enum TezedgeCommitLogError {
```

Comment:
@mambisi 
do we need both: TezedgeCommitLogError and CommitLogError?

Reply:
We dont but we might want to change the implementation in the future

Reply:
@mambisi 
ok, cool, so I would simplify it now, and if there will come any needs, we will change it in future